pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
 	_, err := o.ClientConfig.ClientConfig()
 	return err == nil || !clientcmd.IsNoServerFound(err)
 }
+
+func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
```

Comment:
Pull this code out and put it someplace generic to config instead of being tied to login options.  

> On Apr 19, 2015, at 4:46 PM, Fabiano Franz <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/loginoptions.go:
> 
> > @@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
> >  	_, err := o.ClientConfig.ClientConfig()
> >  	return err == nil || !clientcmd.IsNoServerFound(err)
> >  }
> > +
> > +func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
> There is RelativizeMasterConfigPaths but it takes a different argument (MasterConfig) while we take a *clientcmdapi.Config here, don't think we can reuse it. Apart that, we are sharing MakeAbs and MakeRelative between both functions.
> 
> —
> Reply to this email directly or view it on GitHub.
>